server/routes/manage.ts

```diff
@@ -130,6 +130,7 @@ export default function routes(controllers: Controllers, router: Router, service
       },
     ],
   })
+  get(paths.lostBeds.cancel.pattern, lostBedsController.cancel(), { auditEvent: 'CANCEL_LOST_BED' })
```

Comment:
This gives me the ick a little bit, but I can see why you've done it 😃

Reply:
An alternative might be to check for the presence of the cancel button in the update action and then call the cancellation endpoint instead. Would mean we only need one controller endpoint then too!

Reply:
Yeah I did think about this it was 50/50 but then I thought it might make auditing a bit harder, but could we use the bodyParams to send the value of the cancel button?

Reply:
Ah good point - yeah using the bodyParams should work 👍